tests/TESTING_ROADMAP.md

```
@@ -0,0 +1,173 @@
+  - Complex dependency chains
+
+### To reach higher coverage, the plugin-tests framework needs:
+1. `plugin()` function mock - would enable 4 more functions
+2. `$_SESSION` mock - would enable `debug()`
+3. Docker class integration with DockerMock - would enable `getAllInfo()`
+4. Output buffering test helpers - would enable `postReturn()`
+./bin/phpunit --coverage-text
+
+# Specific test file
+./bin/phpunit tests/unit/PureFunctionsTest.php
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)